### PR TITLE
Phase1 Pixel DQM Plot Changes

### PR DESCRIPTION
**Pull request by schneiml**
Some larger changes to the Phase1 DQM to bring the selection of plots closer to what was agreed on with the pixel experts. This reqired also implementing some missing features and fixing some bugs. (also @fioriNTU @davidcarbonis )
- Allow creating acutal `TProfile`s in step1. 
- Also allow that in 2D.
- Move larger parts of the specifications into a central place, to make it easier to change "defaults". This is pure python magic.
- Slightly change the semantics of specifications involving counts and time on one axis. This leads to a more useful output that could not be specified outherwise, but the definition is a bit vague.
- [Update] also adds the error bars in the profile-like plots from harvesting.
- [Update 2] Fix a bug in the trend plots that potentially also affected others.

### CODE REVIEW

File: DQM/SiPixelPhase1Common/src/HistogramManager.cc [hunk 1]
```diff
@@ -676,6 +681,7 @@ void HistogramManager::executeExtend(SummationStep& step, Table& t, bool isX) {
             // TODO Error etc.?
             new_histo.th1->SetBinContent(i, new_histo.count,
                                          th1->GetBinContent(i, j));
+	    new_histo.th1->SetBinError   (i, new_histo.count, th1->GetBinError(i, j));
```
Comment (dmitrijus) [inline DQM/SiPixelPhase1Common/src/HistogramManager.cc]: you missed a space! ;)

**Comment by dmitrijus** [comment]
+1

File: DQM/SiPixelPhase1Common/src/HistogramManager.cc (hunk 1 above)
Comment (schneiml) [inline DQM/SiPixelPhase1Common/src/HistogramManager.cc]: I blame @davidcarbonis .

Comment (davidcarbonis) [inline DQM/SiPixelPhase1Common/src/HistogramManager.cc]: Guilty as charged.

**Comment by schneiml** [comment]
Since the tests failed due to DAS anyways, I took the opportunity to sneak in a bugfix and a bit of cleanup.

@dmitrijus can you have a look and trigger the tests again? (once DAS is fixed maybe...)

**Comment by dmitrijus** [comment]
please test

**Comment by dmitrijus** [comment]
+1

**Pull request merged**